os/board/rtl8730e/src/component/soc/amebad2/fwlib/ram_hp/ameba_codec.c

```diff
@@ -2013,19 +2013,19 @@ void AUDIO_CODEC_SetMicBstChnMute(u32 amic_sel, u32 type, u32 newstate)
 		} else if (amic_sel == AMIC3) {
 			AUD->AUD_MICBST_CTL1 &= ~AUD_MASK_MICBST2_MUTE_L;
 			if (type == LINEIN) {
-				//RTK_LOGE(TAG, "AMIC3 NOT SUPPORT LINEIN MODE\n");
+				RTK_LOGE(TAG, "AMIC3 NOT SUPPORT LINEIN MODE\n");
```

Comment:
As far as I know, we don't use Realtek Audio. Because of that, we removed audio region in the RAM layout. Please confirm again whether this is necessary or not.

Reply:
This is because some PINs are shared with Audio and are default configurated as Audio functions, so some Audio APIs are needed to disable the Audio PINs. (to enable to work as normal GPIO)
Only a few of the APIs are needed, compiler optimization will exclude the rest.

Reply:
That means this file is built? And audio is not used? Then how about removing the log you added? We need to optimize flash usage. Please check those logs are necessary or not.

Reply:
This file is built, but only a few APIs are included in build, the rest of the APIs are excluded. 
This is to be consistent with our default SDK, so those APIs are remain.

Reply:
I think it's valid with the flat build. But It could be no in the loadable build. Could you verify that almost APIs are excluded in the loadable build?

Reply:
I have checked with laodable_apps_16M config, only 4 Audio APIs are included in the build, other unused APIs have been excluded from the build.